packages/components/src/FormField/FormField.css

```
@@ -69,6 +69,12 @@
   --field--padding-bottom: var(--space-small);
 }
 
+.miniLabel.large {
+  --field--padding-top: calc(
+    var(--space-base) + var(--space-smaller) + var(--space-small)
```

```suggestion
    var(--space-large) + var(--space-smaller)
```

should do the trick. no?